Ensure directories exist

* Prevent assets from being copied to `/opt/dist` instead `/opt/dist/<asset>`.
* Prevents install scripts from changing directories to non existing locations.
Index: emcomm-tools-os-community/source/scripts/bootstrap.sh
===================================================================
--- emcomm-tools-os-community.orig/source/scripts/bootstrap.sh
+++ emcomm-tools-os-community/source/scripts/bootstrap.sh
@@ -5,6 +5,8 @@
 # Purpose : Minimal bootstrap script for the installer
 
 [ ! -d $ET_HOME/bin ] && mkdir -v -p $ET_HOME/bin
+[ ! -d $ET_DIST_DIR ] && mkdir -v -p $ET_DIST_DIR
+[ ! -d $ET_SRC_DIR ] && mkdir -v -p $ET_SRC_DIR
 
 cp -v ../overlay/$ET_HOME/bin/et-log $ET_HOME/bin/
 [ ! -e /usr/local/bin/et-log ] && ln -v -s $ET_HOME/bin/et-log /usr/local/bin/et-log
